docs/metrics/index.md

```diff
@@ -86,6 +86,13 @@ biases, and its intended uses.
     emphasizing larger errors more significantly than smaller ones. It is mainly different from
     [MAE](mean-absolute-error.md) in that larger errors are penalized more heavily.
 
+- [Pearson Correlation Coefficient](pearson-correlation.md)
+
+    ---
+
+    Pearson's correlation coefficient is a statistical measure that evaluates the strength and
```

Comment:
We should be consistent — is it "Pearson Correlation" or "Pearson's Correlation"?

Reply:
quick google looks like `Pearson Correlation` is used by wiki and scipy.